### ISSUE TITLE
[Bug] Moving Token with changed elevation returns validations error "elevation: must be a finite number".

### ISSUE THREAD

**Issue by QSbun** [caewok/fvtt-elevation-ruler#144]
When moving a token and changing the elevation during the ruler measurement, it doesn't change the elevation, instead returning only an error. 

Screenshot of error:
![grafik](https://github.com/user-attachments/assets/b0e20031-94e0-4a02-b332-c572f58526e3)

**Comment by caewok** [comment]
What do you mean "moving a token and changing the elevation during the ruler measurement"? Could you describe the steps you take that cause this?

**Comment by QSbun** [comment]
Ah, apologies

It seems to have been fixed by now? Might have just been a issue on my end which updating fixed...still odd that it happened.

What happened was that when I dragged a token, and changed the elevation with a shortcut, it would when placing the token not change the elevation after. 

1. Select Token 
2. Change elevation up or down with the shortcuts while moving
3. Move Token
4. Error appears, and elevation didn't change.

But as said, is fixed mysteriously now anyway, sorry!!
